pkg/graph/resolvers/mto_milestone_test.go

```diff
@@ -581,3 +581,31 @@ func (suite *ResolverSuite) TestMTOMilestoneNoLinkedSolutions_MultiplePlans() {
 		verifyFunc,
 	)
 }
+
+// This test ensures that the assigned_to field can be set and retrieved correctly
+func (suite *ResolverSuite) TestMTOMilestoneAssignedTo() {
+	// Create model plan
+	plan := suite.createModelPlan("testing milestone assigned_to")
+	assignedTo := uuid.MustParse("00000001-0001-0001-0001-000000000001") // Mint user ID
+
+	// Create a custom milestone with an assigned_to
+	milestone, err := MTOMilestoneCreateCustom(suite.testConfigs.Context, suite.testConfigs.Logger, suite.testConfigs.Principal, suite.testConfigs.Store, "Test Milestone", plan.ID, nil)
+	suite.NoError(err)
+	suite.NotNil(milestone)
+
+	changes := map[string]interface{}{
+		"assignedTo": assignedTo,
+	}
+	updatedMilestone, err := MTOMilestoneUpdate(suite.testConfigs.Context, suite.testConfigs.Logger, suite.testConfigs.Principal, suite.testConfigs.Store, nil, nil, email.AddressBook{}, milestone.ID, changes, nil)
+	suite.NoError(err)
+	suite.NotNil(updatedMilestone)
+	suite.NotNil(updatedMilestone.AssignedTo)
```

Comment:
if you want to avoid the panic, you could do
```suggestion
	if suite.NotNil(updatedMilestone.AssignedTo) {
	   suite.Equal(assignedTo, *updatedMilestone.AssignedTo)
	}
```

One can also override the suite to force this behavior (happens in EASi), but we don't do it in MINT currently.